<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Labelling Application</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget_classes">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>110</y>
      <width>541</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="shapes" name="label_DisplayImage">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>601</width>
      <height>421</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_images">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>601</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>230</y>
      <width>531</width>
      <height>351</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QSpinBox" name="spinBox_RectHeight">
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>360</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QComboBox" name="comboBox_ChooseShape">
       <item>
        <property name="text">
         <string>Triangle</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Rectangle</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Square</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Trapezium</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Polygon</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="comboBox_SetColour">
       <item>
        <property name="text">
         <string>red</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>black</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>blue</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>yellow</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>gray</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>magenta</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>white</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>green</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Pen Width:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Polygon Verticies:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Rectangle Length &amp; Width</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinBoxVerticies">
       <property name="minimumSize">
        <size>
         <width>42</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>42</width>
         <height>22</height>
        </size>
       </property>
       <property name="minimum">
        <number>5</number>
       </property>
       <property name="maximum">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Colour:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="spinBoxPenWidth">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>40</number>
       </property>
       <property name="value">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QSpinBox" name="spinBox_RectWidth">
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>400</number>
       </property>
       <property name="value">
        <number>200</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Square Length &amp; Width</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="spinBox_Square">
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>360</number>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="pushButton_Help">
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="pushButton_HelpSquare">
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>10</y>
      <width>531</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLineEdit" name="lineEdit_className"/>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="CreateClassFile">
       <property name="text">
        <string>Create Class File</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="but_sortClass_Ascending">
       <property name="text">
        <string>Sort Ascending</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="but_sortClass_Descending">
       <property name="text">
        <string>Sort Descending</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="but_CreateClass">
       <property name="text">
        <string>Create Class</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="but_RemoveClass">
       <property name="text">
        <string>Remove Class</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="classBrowse">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_addtext">
       <property name="text">
        <string>Add Text</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>601</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QComboBox" name="comboBoxImages"/>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="but_openImage">
       <property name="text">
        <string>Choose Image</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="checkBox_Date">
       <property name="text">
        <string>Sort by File Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="checkBox_File">
       <property name="text">
        <string>Sort by Date</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="sort_Descending">
       <property name="text">
        <string>Sort Descending</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="sort_Ascending">
       <property name="text">
        <string>Sort Ascending</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>590</y>
      <width>322</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="SaveImageFile">
       <property name="text">
        <string>Save Image File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="AnnotatedFiles">
       <property name="text">
        <string>Browse Annotated Files</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>shapes</class>
   <extends>QLabel</extends>
   <header location="global">shapes.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
